src/Database/Pgsql/composer.json

```diff
@@ -0,0 +1,31 @@
+{
+    "name": "tenancy/db-driver-pgsql",
+    "description": "The tenancy/tenancy database driver for pgsql",
+    "keywords": ["database", "tenancy", "pgsql", "postgresql"],
+    "license": "MIT",
+    "require": {
+        "tenancy/framework": "*",
+        "tenancy/hooks-database": "*",
+        "tenancy/affects-connections": "*",
+        "doctrine/dbal": "^2.9"
```

Comment:
Does it really need this? Might still be abit sleepy, but did not see anything that required dbal

Reply:
That's a good question... that was there from my original work and I honestly can't remember why it was added.  I'll test without it tomorrow and see if it breaks